lips/lip-1.mdx

```diff
@@ -503,7 +503,7 @@ Once the recipient side is comfortable with the KYC evaluation and is ready for
 metadata = TravelRuleMetadatav0 {
   off_chain_reference_id: "123abc", /* The off-chain reference ID specified by `reference_id` in the Off-chain APIs */
 };
-receiver_lcs_data = lcs(metadata, sender_address, amount, "@@$$LIBRA_ATTEST$$@@" /*ASCII-encoded string*/);
+receiver_lcs_data = lcs(metadata, sender_address /*raw binary address*/, amount, "@@$$LIBRA_ATTEST$$@@" /*ASCII-encoded string*/);
```

Comment:
If this has caused confusion before, might be worth adding a bit more detail. E.g., "on-chain account address" + link to https://github.com/libra/lip/blob/master/lips/lip-5.md#terminology